content/docs/reactivesearch/vue/advanced/SelectedFilters.md

```diff
@@ -47,6 +47,9 @@ Example uses:
     Sets the label for the clear all button.
 -   **title** `string` [optional]
     Can be used to set a title
+-   **resetToDefault** `boolean` [optional]
+    When set to true and clearAll functionality is utilised, then it would set the filter's value to its default set value instead of null/ undefined.
```

Comment:
```suggestion
    When set to true and clearAll functionality is utilised, then it would set the filter's value to its default set value(the `defaultValue` prop) instead of null/ undefined.
```